server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -92,6 +94,11 @@ public class RemoteFsTranslog extends Translog {
     private final Semaphore syncPermit = new Semaphore(SYNC_PERMIT);
     private final AtomicBoolean pauseSync = new AtomicBoolean(false);
 
+    // Indicates whether the translog should be uploaded using the new flow, where the checkpoint file is treated as object metadata
+    // associated with the translog file.
+    private final boolean startUploadingTranslogCkpAsMetadata;
+    private final Supplier<Version> minNodeVersionSupplier;
```

Comment:
Do you require these 2 fields to instance variable?

Reply:
Removing `minNodeVersionSupplier`